frontend/src/main/java/com/google/tripmeout/frontend/service/PlaceServiceModule.java

```diff
@@ -0,0 +1,14 @@
+package com.google.tripmeout.frontend.service;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+import com.google.maps.GeoApiContext;
+
+public class PlaceServiceModule extends AbstractModule {
```

Comment:
Consider renaming this module to reflect the fact that it's binding the real maps API service (rather than, e.g., some sort of in-memory fake.